biloba/service.py

```diff
@@ -198,7 +198,10 @@ def join(self):
         # start is idempotent
         self.start()
 
-        self._run_thread.get()
+        try:
+            self._run_thread.get()
```

Comment:
How can this return without `stop_service()` already being called on https://github.com/njoyce/biloba/pull/2/files#diff-5ece57a5fd56a743678b7b75006d5342R96